src/extensions/auto-refresh/bootstrap-table-auto-refresh.js

```diff
@@ -0,0 +1,85 @@
+/* global jQuery:true */
+
+/**
+ * @author: Alec Fenichel
+ * @webSite: https://fenichelar.com
+ * @version: v1.0.0
+ */
+
+(function ($) {
+
+  'use strict';
+
+  $.extend($.fn.bootstrapTable.defaults, {
+    autoRefresh: false,
+    autoRefreshInterval: 60,
+    autoRefreshSilent: true,
+    autoRefreshStatus: true,
+
```

Comment:
This was done intentionally to distinguish between the "public" and "private" variables. I have removed the line.